staging/prometheus-operator/templates/alertmanager/psp.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: {{ template "prometheus-operator.fullname" . }}-alertmanager
-  namespace: {{ $.Release.Namespace }}
```

Comment:
namespace not needed? This would be installed in kubeaddons in helm3 is this intentional?

Reply:
Looks like they took it out upstream: https://github.com/helm/charts/blob/master/stable/prometheus-operator/templates/alertmanager/psp.yaml#L5-L7

Do we need it back? If so, I can add another patch to the set in PR #742